### PR TITLE
add unitegallery v1.7.18 with git autoupdate config

### LINKED ISSUE #5373: [Request] Add  library Unite Gallery
**Library name: Unite Gallery** 
**Git repository url: https://github.com/vvvmax/unitegallery/**
**Official homepage: http://unitegallery.net/**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### LINKED ISSUE #7032: [Request] Unite Gallery
**Library name: Unite Gallery** 
**Git repository url: https://github.com/vvvmax/unitegallery/**
**npm package url(optional):** 
**License(s):**
**Official homepage: http://unitegallery.net/**
**Wanna say something? Leave message here: Best slider available**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #7032 
@x09326 Please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/vvvmax/unitegallery/
 * Official website (optional, not the repository): http://unitegallery.net/
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 196
   - Count of watchers: 26
   - Count of forks: 47
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #5373, close #7032, cc @vvvmax

### CODE REVIEW

File: ajax/libs/unitegallery/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "jQuery",
+    "Responsive"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/vvvmax/unitegallery.git",
+    "basePath": "package/unitegallery",
+    "files": [
+      "**/*.!(txt)"
+    ]
```
Comment (x09326) [inline ajax/libs/unitegallery/package.json]: Maybe you can discuss with its author and see if it's possible add a dist folder for release files?
I think this is similar to my case #7604.

**Comment by pvnr0082t** [comment]
@x09326 I have update ```package.json``` file and the latest version is v1.7.17. Please help me review again, thank you.

**Comment by x09326** [comment]
@PeterDaveHello 
I think this PR is okay, please help me review it, thanks.

**Comment by pvnr0082t** [comment]
I have done the rebase upon the master branch. ping @PeterDaveHello. :+1:

**Comment by PeterDaveHello** [comment]
Oops, sorry that I missed this PR.

**Comment by PeterDaveHello** [comment]
@pvnr0082t usually updated PR need another fresh review.

**Comment by pvnr0082t** [comment]
@x09326 I have updated this pull request to v1.7.18. Please help me review it again, thank you.

**Pull request merged**